best_practices/controllers.rst

```diff
@@ -106,7 +106,7 @@ for the homepage of our app:
         public function indexAction()
         {
             $posts = $this->getDoctrine()
-                ->getRepository('AppBundle:Post')
+                ->getRepository(Post::class)
```

Comment:
With this change, we'll need to make sure that the correct `use` statement is at the top of each code block.